scripts/simulator_base.js

```diff
@@ -2693,12 +2693,6 @@ var SIMULATOR = {};
 					if (debug) {
 						echo += debug_name(current_assault) + ' recovers from corrosion<br>';
 					}
-				} else {
-					var corrosion = corroded.amount;
-					current_assault.attack_corroded = corrosion;
-					if (debug) {
```

Comment:
We'll still want this code, even if just for the logging.  Setting `current_assault.attack_corroded` here is minimal overhead and ensures that we always end each turn with the correct value.  Technically, it's not needed, as `attack_corroded` should only be getting modified when we either add more or the timer runs out.